app/Http/Controllers/External/VatsimNet/ProcessVatsimNetWebhook.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Controllers\External\VatsimNet;
+
+use App\Http\Controllers\BaseController;
+
+class ProcessVatsimNetWebhook extends BaseController
+{
+    public function __invoke()
+    {
+        if (request()->header('Authorization') !== config('services.vatsim-net.webhook.key')) {
+            return response()->json([
+                'status' => 'forbidden',
+            ], 403);
+        }
+
+        foreach (request()->json('actions') as $action) {
+            $class = config("services.vatsim-net.webhook.jobs.{$action['action']}");
+            if ($class && class_exists($class)) {
+                dispatch(new $class(request()->json('resource'), $action))->afterResponse();
```

Comment:
for ease of running sync locally here can we just drop `->afterResponse()` entirely. Dispatching queue jobs should be fast anyway and I don't think the system sending the webhooks is gonna mind too much

```suggestion
                dispatch(new $class(request()->json('resource'), $action));
```

Reply:
->defer() actually handles the sync driver proper and waits until after the response is returned too. So probably best of both worlds to do that.